### PR TITLE
Adding limit query parameter origin inspector

### PR DESCRIPTION
**Pull request by jsonroy-fastly**
This PR adds the missing `limit` query parameter to the request on origin inspector endpoint.

### CODE REVIEW

File: fastly/stats_origin_inspector.go [hunk 1]
```diff
@@ -91,6 +91,8 @@ type GetOriginMetricsInput struct {
 	End *time.Time
 	// GroupBy is the dimensions to return in the query.
 	GroupBy []string
+	// Limit is the limit of returned data
```
[the remark this answers is not in the document]
Comment (jsonroy-fastly) [reply fastly/stats_origin_inspector.go]: I changed it to `Limit is the limit of entries requested`, I also took the initiative to change it under domain inspector too.

File: fastly/stats_domain_inspector.go [hunk 2]
```diff
@@ -117,7 +117,7 @@ type GetDomainMetricsInput struct {
 	End *time.Time
 	// GroupBy is the dimensions to return in the query.
 	GroupBy []string
-	// Limit is the limit of returned data
+	// Limit is the limit of entries requested
```
Comment (kpfleming) [inline fastly/stats_domain_inspector.go:120]:
```suggestion
	// Limit specifies the maximum number of entries to be returned
```

File: fastly/stats_origin_inspector.go [hunk 3]
```diff
@@ -91,6 +91,8 @@ type GetOriginMetricsInput struct {
 	End *time.Time
 	// GroupBy is the dimensions to return in the query.
 	GroupBy []string
+	// Limit is the limit of entries requested
```
Comment (kpfleming) [inline fastly/stats_origin_inspector.go:94]:
```suggestion
	// Limit specifies the maximum number of entries to be returned
```

**Review by kpfleming — APPROVED** [review]
I edited the 'limit' descriptions to be more in line with other descriptions across our API schemas.

**Pull request merged**